ecosystem/node-checker/src/metric_evaluator/build_evaluators.rs

```diff
@@ -38,20 +35,5 @@ pub fn build_evaluators(
         )));
     }
 
-    let in_use_evaluators_names = evaluators
-        .iter()
-        .map(|e| e.get_name())
-        .collect::<HashSet<_>>();
-    for evaluator_string in evaluator_strings {
```

Comment:
We don't want to check if they've given us an unknown evaluator string? Or is this no longer possible?